### DISCUSSION TITLE
How to update/reset user's password via REST API?

### DISCUSSION THREAD

**Discussion by zolghadri** [keycloak/keycloak#21911] [Q&A] [answered]
I'm new in Keycloak and i want users to be able to change or reset their password by REST API.
I found `https://site.com/{{realm}}/users/{{user_id}}/reset-password` but i got  the below message  .
```
{
    "error": "RESTEASY003210: Could not find resource for full path: https://site.com/therealm/users/user-id/reset-password"
}
```

I want to know any solution to update/reset password via REST API.
**Thank you for help**

**Comment by danielFesenmeyer** [comment, marked as answer]
The correct url is probably `https://site.com/**admin**/{{realm}}/...`
